### PR TITLE
Adds E-fuels, oh and a few other elzu changes relating to blood and APCs-unstale edition!

### PR DESCRIPTION
**Pull request by Sylphily**
## About The Pull Request

changes some stuff for elzu, making it so that they can drain more power from apcs, and adds the NEW AND SHOCKING E-Fuel, which charges them!, also adds their blood back

## Why It's Good For The Game

Discord_1sxh7XvtzU

## Changelog

:cl:
add: E-Fuel
tweak: elzu now need their own blood now!!
tweak: elzu can now drain more power from APCs
/:cl:

### CODE REVIEW

**Comment by tiramisuapimancer** [comment]
hold on, going to try making a review and most of the contents are gonna be petty, not sure how github reviews work either

File: code/modules/reagents/chemistry/reagents/food_reagents.dm [hunk 1]
```diff
@@ -720,24 +720,30 @@
 
 /datum/reagent/consumable/liquidelectricity
 	name = "Liquid Electricity"
-	description = "The blood of Ethereals, and the stuff that keeps them going. Great for them, horrid for anyone else."
+	description = "The blood of Elzuose, and the stuff that keeps them going. Great for them, horrid for anyone else."
```
Comment (tiramisuapimancer) [inline code/modules/reagents/chemistry/reagents/food_reagents.dm]:
```suggestion
	description = "The blood of elzuosa, and the stuff that keeps them going. Great for them, horrid for anyone else."
```

File: code/modules/reagents/chemistry/reagents/food_reagents.dm [hunk 2]
```diff
@@ -720,24 +720,30 @@
...
-
 /datum/reagent/consumable/liquidelectricity/on_mob_life(mob/living/carbon/M)
 	if(prob(25) && !isethereal(M))
 		M.electrocute_act(rand(10,15), "Liquid Electricity in their body", 1) //lmao at the newbs who eat energy bars
 		playsound(M, "sparks", 50, TRUE, SHORT_RANGE_SOUND_EXTRARANGE)
 	return ..()
 
+/datum/reagent/consumable/electrolytes
+	name = "Electrolytes"
+	description = "its what plants crave, and lightbulbs"
```
Comment (tiramisuapimancer) [inline code/modules/reagents/chemistry/reagents/food_reagents.dm]:
```suggestion
	description = "It's what crystal-plant-dragons crave."
```

**Comment by Sylphily** [comment]
well that was easy to change, literal press of a single button

**Comment by tiramisuapimancer** [comment]
👍

**Comment by ZephyrTFA** [comment]
Do not open the exact same PR again because the old one went stale. Ping a maintainer on github or the discord.

**Comment by Sylphily** [comment]
> Do not open the exact same PR again because the old one went stale. Ping a maintainer on github or the discord.

bruh i was literally told to open a new one
image

**Pull request closed without merging**